gui/default/assets/css/overrides.css

```
@@ -143,7 +143,7 @@ table.table-auto td {
     max-width: 0px;
 }
 
-td input[type="checkbox"] {
+table > tbody > tr > td.extended-attributes-filter-rule-checkbox input[type="checkbox"] {
```

Why not just

```suggestion
.extended-attributes-filter-rule-checkbox input[type="checkbox"] {
```

?

The class name is already super specific 🙂.

---

Yeah, but not specific enough in CSS terms :smile: Bootstrap already ships with a style that overrides this:

```css
.table > thead > tr > th, .table > tbody > tr > th, .table > tfoot > tr > th, .table > thead > tr > td, .table > tbody > tr > td, .table > tfoot > tr > td {
    vertical-align: top;
    /* other styles... */
}
```

...ah right, but that was when I was testing out replacing the `<input> margin-top` with a `<td> vertical-align` instead. I decided to not do both those changes at the same time, to make sure I don't break something in some particular browser or something.

The `<input> margin-top` clashes with a similar Bootstrap override, but you're quite right that at least the parent selectors are not needed for this!

---

Done in 3eb382a75862d790513a65de322595686a348140.

---

I forgot how nasty those Bootstrap selectors can be 🙃.